### DISCUSSION TITLE
How should we process chunks loaded before the installation of this mod?

### DISCUSSION THREAD

**Discussion by IAmREGE** [Felicis/Water-Physics-Overhaul#55] [Ideas]
Currently (and maybe FINAL I think :smile:), Sasai's behaviour is to keep the original flowing fluid blocks(including `falling=true`) unchanged and to tick these block once they received BUD as a regular physics fluid block. So you could:
1. Without the mod installed, place a bucket of water on default Superflat;
2. Re-enter the world with the mod installed;
3. You will get 43 buckets of water.

Should we change it?

**Comment by SKWID011** [comment]
Don't think this would really affect me as I'll be starting a new survival once the update for 1.18 comes out that fixes the y level problems. Though I do question the 43 buckets of water. Why 43?
